transcripts/3.5 My Virtual Environments Workflow.txt

```diff
@@ -29,12 +29,12 @@
 01:51 is it's going to deactivate the previous virtual environment 
 01:54 and then activate the new one. 
 01:56 So I don't have to type deactivate first, and then of course,
-02:00 I could do m work and just deactivate the virtual environment again. 
+02:00 I could do my work and just deactivate the virtual environment again. 
 02:04 So I am using these ae and de shortcuts or aliases to make it a little bit easier 
 02:09 to work with virtual environments from the command line. 
 02:12 And now I am going to show you how those work behind the scenes. 
 02:15 So what I have done here is I've modified the config file from my shell, 
-02:20 I am using bash so this would be bash profile. 
+02:20 I am using bash so this would be ~/.bash_profile
```

Comment:
@TeddyBear06 this was a nice catch. These kinds of changes definitely take things to the next level. ;)